.github/workflows/test.yml

```diff
@@ -129,6 +133,16 @@ jobs:
           build:asan --config=limit-storage
           build:debug --config=limit-storage
           EOF
+      - name: Fix macOS fastbuild configuration
+        # Unlike the bazel Unix toolchain the macOS toolchain sets "-O0 -DDEBUG" for fastbuild by
+        # default. This is unhelpful for compile speeds and test performance, remove the DEBUG
+        # define.
+        if: ${{ !(contains(matrix.config.suffix, 'debug') || contains(matrix.config.suffix, 'asan')) }}
```

Comment:
Why is this condition necessary? I thought we were making it so that macos didn't have debug or asan configs?

Reply:
You're right, with this PR both the debug and asan macOS jobs will be disabled. I think it's future proof in case we enable these runners again in the future though. The flags themselves are being added here https://github.com/bazelbuild/apple_support/blob/facfb550e423d2a8b8c7835bd31e9d1d6af51f14/crosstool/cc_toolchain_config.bzl#L2024